src/components/Menu/Menu.module.css

```diff
@@ -10,17 +10,19 @@
 }
 
 .menu__popover {
-  max-width: var(--eds-l-max-width);
   margin-top: var(--eds-size-1);
+  min-width: max-content;
+  max-width: var(--eds-l-max-width);
   position: absolute;
+  width: 100%;
 }
 
-.menu__button {
+.menu__button.menu__button {
   background-color: var(--eds-theme-color-form-background);
 }
 
 /* Needed to create higher specificity than ClickableStyle. TODO: improve */
-.menu__button--icon.menu__button--icon.menu__button--icon {
+.menu__button--icon.menu__button--icon.menu__button--icon.menu__button--icon {
```

Comment:
oof

Reply:
Oof indeed. I think we need to revisit how this clickablestyle thing works, as it creates a lot of specificity challenges :S